src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PLPController.kt

```diff
@@ -51,6 +52,27 @@ class PLPController(
     return DataResponse(response.data)
   }
 
+  @GetMapping("{hmppsId}/plp-induction-schedule/history")
+  @Operation(
+    summary = "Returns plp the induction schedule history associated with a person.",
+    responses = [
+      ApiResponse(responseCode = "200", useReturnTypeSchema = true, description = "Successfully found induction schedule history for a person with the provided HMPPS ID."),
+      ApiResponse(responseCode = "404", content = [Content(schema = Schema(ref = "#/components/schemas/PersonNotFound"))]),
+      ApiResponse(responseCode = "500", content = [Content(schema = Schema(ref = "#/components/schemas/InternalServerError"))]),
+    ],
+  )
+  fun getInductionScheduleHistory(
+    @Parameter(description = "A HMPPS id", example = "A123123") @PathVariable hmppsId: String,
+  ): DataResponse<InductionSchedules> {
+    val response = getInductionScheduleForPersonService.getHistory(hmppsId)
+
+    if (response.hasError(UpstreamApiError.Type.ENTITY_NOT_FOUND)) {
+      throw EntityNotFoundException("Could not find person with id: $hmppsId")
+    }
+    auditService.createEvent("GET_INDUCTION_SCHEDULE", mapOf("hmppsId" to hmppsId))
```

Comment:
I'm not sure how these events work / how strongly typed there are etc; but should the event name be plural perhaps? (or maybe it doesnt matter so much? 🤷‍♂️ )

Reply:
it's not typed at all, nor is there any guidance. I was thinking of keeping it simple with only considering INDUCTION or REVIEW schedules but I've updated this one to have _HISTORY